pyproject.toml

```diff
@@ -23,7 +23,7 @@ ignore_errors = true
 # when the library is converted, the above entry will be replaced (strict is demanded)
 [[tool.mypy.overrides]]
 module = [
-    'arelle.plugin.validate.ESEF',
+    'arelle.plugin.validate.ESEF.*',
```

Comment:
I made a mistake here. Do you want this, and related changes, to go in a separate PR? I believe the changes would be this line and the line edited in `Util.py`.

Reply:
No, this is fine, it's all ESEF and type related. You can keep it together.

Reply:
OK, thanks! I'll leave it as a draft until the other one.